src/pages/Share/ShareTab.tsx

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import {saveUnknownUserDetails} from '@libs/actions/Share';
+import Navigation from '@libs/Navigation/Navigation';
+import MoneyRequestParticipantsSelector from '@pages/iou/request/MoneyRequestParticipantsSelector';
+import CONST from '@src/CONST';
+import ROUTES from '@src/ROUTES';
+
+function ShareTab() {
+    return (
+        <MoneyRequestParticipantsSelector
+            iouType={CONST.IOU.TYPE.SUBMIT}
+            onParticipantsAdded={(value) => {
+                const participant = value.at(0);
```

Comment:
This code repeats almost everything from `SubmitTab`, maybe we want to reuse it?